app/server/appsmith-server/src/test/java/com/appsmith/server/dtos/PaginationDTOTest.java

```diff
@@ -0,0 +1,23 @@
+package com.appsmith.server.dtos;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PaginationDTOTest {
+
+    @Test
+    void getTotalPage() {
+        PaginationDTO paginationDTO = new PaginationDTO(0, 10, 100);
+        assertEquals(10, paginationDTO.getTotalPage());
```

Comment:
Can we use `assertThat` method of assertions? They tend to produce better error messages and it also helps us be consistent across our tests.

```suggestion
        assertThat(paginationDTO.getTotalPage()).isEqualTo(10);
```

Reply:
I'll do it.